### PR TITLE
Migrate composer action menu icons to material design icons + migrate to Vue buttons

### PR DESCRIPTION
**Pull request by GretaD**
ref https://github.com/nextcloud/groupware/issues/38

### CODE REVIEW

File: src/components/Composer.vue [hunk 1]
```diff
@@ -234,17 +234,26 @@
...
-						<ActionButton icon="icon-upload" @click="onAddLocalAttachment">
+						<ActionButton @click="onAddLocalAttachment">
+							<template #icon>
+								<IconUpload :size="20" />
+							</template>
 							{{
 								t('mail', 'Upload attachment')
 							}}
 						</ActionButton>
 						<ActionButton icon="icon-folder" @click="onAddCloudAttachment">
```
Comment (ChristophWurst) [inline src/components/Composer.vue:245]: Icon still used

**Review by ChristophWurst — APPROVED** [review]

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Why did you rename `button` to `Button`? the nc/vue button component is imported as `VButton`

Are the button changes even necessary for the icon replacement?

**Comment by GretaD** [comment]
> Why did you rename `button` to `Button`? the nc/vue button component is imported as `VButton`
> 
> Are the button changes even necessary for the icon replacement?

no there are not necessary on this pr because <button> doesnt have an icon. When it has an icon then it needs <Button> I did it here because ive done it with other PRs

**Review by ChristophWurst — APPROVED** [review]
:+1: :elephant:

**Comment by raimund-schluessler** [comment]
There is quite a bit of chaos with the buttons now, you have `button`, `Button` and `VButton`, whereas they all should be `VButton` (or how you want to import them) regardless of whether they have icons or not. `button` will only render to a standard HTML component and not profit from the accessibility features of the vue component.

`button` (standard HTML button with an icon even):
https://github.com/nextcloud/mail/blob/57ffa52197d78275ce4a94595128b9b85a92bfe7/src/components/Composer.vue#L61-L67

`Button` (probably also standard HTML with an icon):
https://github.com/nextcloud/mail/blob/57ffa52197d78275ce4a94595128b9b85a92bfe7/src/components/Composer.vue#L355-L363

`Button` without icon:
https://github.com/nextcloud/mail/blob/57ffa52197d78275ce4a94595128b9b85a92bfe7/src/components/Composer.vue#L377-L379

**Pull request merged**